apps/liteprotocoltester/run_tester_node_on_fleet.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+#set -x
+#echo "$@"
+
+if test -f .env; then
+  echo "Using .env file"
+  . $(pwd)/.env
+fi
+
+
+echo "I am a lite-protocol-tester node"
+
+BINARY_PATH=$1
+
+if [ ! -x "${BINARY_PATH}" ]; then
+  echo "Invalid binary path '${BINARY_PATH}'. Failing"
+  exit 1
+fi
+
+if [ "${2}" = "--help" ]; then
```

Comment:
I wonder if we should also show the following help in case no parameter is passed to the script

Reply:
That is the default entry point of the docker image as it expect you to add arguments.